.github/workflows/validator-update.yml

```diff
@@ -0,0 +1,37 @@
+name: Update validation reports after GTFS Validator has been updated
+on:
+  workflow_dispatch: # Supports manual trigger
+  repository_dispatch:
+    types: [
+      gtfs-validator-release, # Triggered by a release from the Canonical GTFS Validator
+      gtfs-validator-update-stg # Triggered by a merge to `master` in the Canonical GTFS Validator
+    ]
+jobs:
+  validator-update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Authenticate to Google Cloud
+        id: gcloud_auth
+        uses: google-github-actions/auth@v2
+        with:
+          credentials_json: ${{ github.event.action == 'gtfs-validator-release' && secrets.PROD_GCP_MOBILITY_FEEDS_SA_KEY || secrets.DEV_GCP_MOBILITY_FEEDS_SA_KEY }}
+
+      - name: GCloud Setup
+        uses: google-github-actions/setup-gcloud@v2
+
+      - name: Set environment variables
+        run: |
+          echo "ENVIRONMENT=${{ github.event.action == 'gtfs-validator-release' && vars.PROD_MOBILITY_FEEDS_ENVIRONMENT || vars.DEV_MOBILITY_FEEDS_ENVIRONMENT }}" >> $GITHUB_ENV
```

Comment:
I must be missing something. I don't see where PROD_MOBILITY_FEEDS_ENVIRONMENT and DEV_MOBILITY_FEEDS_ENVIRONMENT are defined.

Reply:
These are repository level variables

Reply:
Oups. I was looking at secrets.